### PR TITLE
adds timluc-miptev's gall guide to comm. tutorials

### PR DESCRIPTION
**Pull request by drbeefsupreme**
Considering how thorough and helpful this is, and how long its been around, I'm embarrassed it hasn't been linked sooner.

----

#

### CODE REVIEW

**Comment by matildepark** [comment]
I think it's intended to become part of the actual docs, but Josh is
overseeing this I think.

**Comment by drbeefsupreme** [comment]
Yeah gonna close this based on a conversation with @jalehman just now

**Comment by drbeefsupreme** [comment]
@jalehman

File: content/using/develop.md [hunk 1]
```diff
@@ -76,13 +76,13 @@ Now you should see a block of boot messages, starting with the Urbit version num
...
-Here are some tutorials written by members of the Urbit community. Some are a little outdated, but they may still prove useful.
+Here are some tutorials written by members of the Urbit community.
 
-- [Generators](https://github.com/joemfb/mardev/tree/master/docs/gen) by `~master-morzod`
-- [Urbit by Doing](https://github.com/Fang-/Urbit-By-Doing) by `~palfun-foslup`
-- [Learning Hoon](https://github.com/knubie/learning-hoon) by `@knubie`
 - [A Nock Interpreter](https://jtobin.io/nock) by `~nidsut-tomdun`
 - [Basic Hoonery](https://jtobin.io/basic-hoonery) by `~nidsut-tomdun`
+- [The Complete Guide to
+  Gall](https://github.com/timlucmiptev/gall-guide/blob/master/overview.md) by `~timluc-miptev`
```
Comment (jalehman) [inline content/using/develop.md]:
```suggestion
- [The Complete Guide to Gall](https://github.com/timlucmiptev/gall-guide/blob/master/overview.md) by `~timluc-miptev`
```

**Review by jalehman — APPROVED** [review]
What about https://github.com/lukechampine/rote? 

Otherwise, pending one small formatting question, looks good.

**Comment by drbeefsupreme** [comment]
Whoops I forgot to include the author of the rote guide. @lukechampine what is your ship's name?

**Comment by lukechampine** [comment]
I'm `~watter-parter`, thanks!

**Pull request merged**